src/js/components/Grommet/Grommet.js

```diff
@@ -22,11 +34,24 @@ class Grommet extends Component {
     const { theme } = this.props;
 
     return {
-      grommet: {},
+      grommet: {
+        announce: this.announce,
+      },
       theme: deepMerge(baseTheme, theme),
     };
   }
 
+  announce = (message, mode = 'polite') => {
+    const announcer = document.body.querySelector('[aria-live]') || createAnnouncer();
+
```

Comment:
Can you add a comment explaining why we do the following code? It just seems strange when you don't know why.